packages/grafana-prometheus/package.json

```
@@ -1,6 +1,6 @@
 {
   "author": "Grafana Labs",
-  "license": "Apache-2.0",
+  "license": "AGPL-3.0-only",
   "name": "@grafana/prometheus",
   "private": true,
   "version": "0.0.1",
```

This should match the other packages version across Grafana.
```suggestion
  "version": "10.4.0-pre",
```